composer.json

```diff
@@ -28,7 +28,7 @@
         "josegonzalez/dotenv": "2.*",
         "mobiledetect/mobiledetectlib": "2.*",
         "phpoffice/phpspreadsheet": "^1.15",
-        "wikimedia/composer-merge-plugin": "^1.4"
+        "wikimedia/composer-merge-plugin": "^2.0"
```

Comment:
This change is not related to the fix (it's related to the use of composer2)

Reply:
right, but why do we have this diff since in `master` we have already `"wikimedia/composer-merge-plugin": "^2.0.1"` ? :thinking:

Reply:
some merge conflict. resolved with a push...